arches_for_science/settings.py

```diff
@@ -1,348 +0,0 @@
-    "http://purl.org/dc/terms/": "dcterms",
-    "http://purl.org/dc/elements/1.1/": "dc",
-    "http://schema.org/": "schema",
-    "http://www.w3.org/2004/02/skos/core#": "skos",
-    "http://www.w3.org/2000/01/rdf-schema#": "rdfs",
-    "http://xmlns.com/foaf/0.1/": "foaf",
-    "http://www.w3.org/2001/XMLSchema#": "xsd",
-    "https://linked.art/ns/terms/": "la",
-    "http://www.w3.org/1999/02/22-rdf-syntax-ns#": "rdf",
-    "http://www.cidoc-crm.org/cidoc-crm/": "",
-    "http://www.ics.forth.gr/isl/CRMdig/": "",
-    "http://www.ics.forth.gr/isl/CRMgeo/": "geo",
-    "http://www.ics.forth.gr/isl/CRMsci/": "sci",
-}
-
-
-RENDERERS += [
-    {
-        "name": "fors-reader",
-        "title": "ASD Hi Res FieldSpec4",
-        "description": "Use for exports from all our ASD High Resolution Field Spectroscopy",
-        "id": "88dccb59-14e3-4445-8f1b-07f0470b38bb",
-        "iconclass": "fa fa-bar-chart-o",
-        "component": "views/components/cards/file-renderers/fors-reader",
-        "ext": "txt",
-        "type": "text/plain",
-        "exclude": "",
-    },
-    {
-        "name": "xrf-reader",
-        "title": "HP Spectrometer XRF ASCII Output",
-        "description": "Use for exports from all our HP XRF outputs",
-        "id": "31be40ae-dbe6-4f41-9c13-1964d7d17042",
-        "iconclass": "fa fa-bar-chart-o",
-        "component": "views/components/cards/file-renderers/xrf-reader",
-        "ext": "txt",
-        "type": "text/plain",
-        "exclude": "",
-    },
-    {
-        "name": "raman-reader",
-        "title": "Raman File Reader",
-        "description": "Use for exports from all our HP raman and gas chromatograph spectrometers",
-        "id": "94fa1720-6773-4f99-b49b-4ea0926b3933",
-        "iconclass": "fa fa-bolt",
-        "component": "views/components/cards/file-renderers/raman-reader",
-        "ext": "txt",
-        "type": "text/plain",   
-        "exclude": "",
-    },
-    {
-        "name": "pdbreader",
-        "title": "PDB File Reader",
-        "description": "",
-        "id": "3744d5ec-c3f1-45a1-ab79-a4a141ee4197",
-        "iconclass": "fa fa-object-ungroup",
-        "component": "views/components/cards/file-renderers/pdbreader",
-        "ext": "pdb",
-        "type": "",
-        "exclude": "",
-    },
-    {
-        "name": "pcdreader",
-        "title": "Point Cloud Reader",
-        "description": "",
-        "id": "e96e84f2-bcb2-4ca4-8793-7568b09d7374",
-        "iconclass": "fa fa-cloud",
-        "component": "views/components/cards/file-renderers/pcdreader",
-        "ext": "pcd",
-        "type": "",
-        "exclude": "",
-    },
-    # {
-    #     "name": "colladareader",
-    #     "id": "3732bdf0-74b1-412f-955a-9ca038e7db31",
-    #     "iconclass": "fa fa-spoon",
-    #     "component": "views/components/cards/file-renderers/colladareader",
-    #     "ext": "dae",
-    #     "type": "",
-    #    "exclude": [],
-    # },
-]
-
-X_FRAME_OPTIONS = "SAMEORIGIN"
-
-FORMATS = [
```

Comment:
I very much like the idea of shipping a smaller settings.py, but removing the whole thing undoes arches-project/disco@95a5b8aef03b931f244e65549e408462257ece11. Maybe you have a recommendation for a very minimal set of settings we could keep?

Reply:
if possible we should discuss why an arches app would need a settings file? Django apps usually don't and instead require the project to adjust its settings file. And if it's only there to keep testing happy I think it would be best to investigate a way around that.

Reply:
I hear what you're saying. It's not an anti-pattern per se, but I think the way we're including it arches apps should be refactored. The settings file included in your examples are wholly contained to the package, I think that's not entirely possible with arches apps.

What I've noticed from DISCO is that in the project's `settings.py` file, we're no longer importing settings from arches core. That has been moved to the arches-for-science app. To me this is an anti-pattern. Also having multiple arches apps importing their own `settings.py` files into a project could result in some confusing situations around static files, templates, etc.

I'll create a sister ticket in DISCO and update the project's settings file, along with updating `test_settings.py` to function with the changes and `README.md` to log the changes that need to be made a project's settings file.